src/dto/allocation-dto.ts

```diff
@@ -1,6 +1,8 @@
+import { IHoldout } from './holdout-dto';
 import { IVariation } from './variation-dto';
 
 export interface IAllocation {
   percentExposure: number;
   variations: IVariation[];
+  holdout: IHoldout | null;
```

Comment:
I think this would be the first time we'd be introducing nullable fields in the RAC. Any objection to that?

Reply:
Do you remember why in the design doc for the RAC API we had a list of holdouts here?

Reply:
I think that just needed to be cleaned up. I'll make the notion doc consistent with this. The keys field already accommodates multiple holdouts

```
          holdout: {
            statusQuo: 'variation-7',
            shipped: 'variation-8',
            keys: [
              {
                holdoutKey: 'holdout-2',
                statusQuoShardRange: {
                  start: 0,
                  end: 25,
                },
                shippedShardRange: {
                  start: 26,
                  end: 50,
                },
              },
             ... (multiple holdouts in this list)
            ],
           ...
         }
```